core/user/service.go

```diff
@@ -210,3 +212,10 @@ func (s Service) FetchCurrentUser(ctx context.Context) (User, error) {
 
 	return fetchedUser, nil
 }
+
+func (s Service) DeleteUser(ctx context.Context, id string) error {
```

Comment:
```suggestion
func (s Service) Remove(ctx context.Context, id string) error {
```
or
```suggestion
func (s Service) Delete(ctx context.Context, id string) error {
```
No need to call `DeleteUser` since we are already under user domain